### PR TITLE
Upgrade a variety of development tools and utilities to current

### PR DESCRIPTION
**Pull request by CloCkWeRX**
```
$ bundle update selenium-webdriver guard guard-rspec rails factory_girl pg simplecov coveralls newrelic_rpm better_errors minitest
```

None of these should have much of an end user visible impact; unless say the pg gem contains problems.

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
Phwoar, that's a big coverage decrease for simply upgrading coveralls :P

**Comment by maco** [comment]
On note of "automatic checks that get run" (like Coveralls), where I'm
working now uses Code Climate, so I've learned some about what the heck its
results mean, and I'm pondering making an attempt at making it happier.
(Right now, it's so ornery we're just ignoring it) For instance, I've
learned what it wants when it says a function is too complex, and well... I
think I should refactor the photo bit I wrote that's like 7 nested if's
deep.

File: Gemfile.lock [hunk 1]
```diff
@@ -261,27 +257,29 @@ GEM
     letter_opener (1.3.0)
       launchy (~> 2.2)
     libv8 (3.16.14.7)
-    listen (3.0.2)
-      rb-fsevent (>= 0.9.3)
-      rb-inotify (>= 0.9)
-    lumberjack (1.0.9)
+    listen (3.1.4)
```
Comment (pozorvlak) [inline Gemfile.lock]: This won't install for me without also upgrading Ruby to v2.2.

**Comment by pozorvlak** [comment]
OK, I've finally worked out what's going on with the coverage. The lines covered haven't changed, but the new version of coveralls includes files with 0% coverage in the total. This is clearly correct, so we should accept the new, lower figure. Or rather, we should *not* accept it, and write tests covering the files we're missing!

**Comment by maco** [comment]
👍

**Comment by pozorvlak** [comment]
@CloCkWeRX, can you rebase this onto #844 because of the dependency issue I pointed out above?

**Comment by CloCkWeRX** [comment]
Yeah can do (or merging that one first might be quicker, laptop being upset
at me)

**Comment by maco** [comment]
I'm fine with merging that one first. It has a conflict at the moment, though.

**Comment by maco** [comment]
@CloCkWeRX if you resolve the merge conflict on #844 I'll merge it tonight

**Pull request merged**